test/yaml/serialization_test.rb

```diff
@@ -29,8 +29,14 @@ class SerializationTest < Minitest::Test
     assert_equal "control 0x01 char \n whoops!", load_yaml(chr_1_yaml)
   end
 
+  test 'should handle non-binary yaml with escaped things that look like control chars' do
+    # irb> Psych.dump(['\\x01 \\xAF \\\\xAF', "\x01 \\\x01 \x01\\\x01"])
+    escaped_yaml = "---\n- \"\\\\x01 \\\\xAF \\\\\\\\xAF\"\n- \"\\x01 \\\\\\x01 \\x01\\\\\\x01\"\n"
+    assert_equal ['\\x01 \\xAF \\\\xAF', '0x01 \\0x01 0x01\\0x01'], load_yaml(escaped_yaml)
+  end
+
   test 'should leave non-binary JSON with things that look like control chars unchanged' do
-    hash = { 'report' => ' \x09 ' }
+    hash = { 'report' => ' \x01 ' }
```

Comment:
Hi @bshand - what was the reason for this test being changed?

Reply:
I wanted to make it more consistent with the adjacent tests, for clarity. They all use `\x01` as an example of a control character, which `psych` translates to `\x01` in the YAML.

It also reduces confusion, because `\x09` is a tab character, which translates to `\t` in YAML. I couldn't use `\x09` in my new examples because of this.